docs/trace/extending-the-sdk/README.md

```diff
@@ -317,3 +317,5 @@ A demo ResourceDetector is shown [here](./MyResourceDetector.cs).
   specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-processor)
 * [Sampler
   specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#sampler)
+* [Resource
```

Comment:
Minor: sort alphabetically (R before S) unless there is a good reason not to do so.